recipes/marshmallow-dataclass/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set pypi_name = "marshmallow_dataclass" %}
+{% set conda_name = "marshmallow-dataclass" %}
+{% set version = "8.0.0" %}
+
+package:
+  name: "{{ conda_name }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ pypi_name[0] }}/{{ pypi_name }}/{{ pypi_name }}-{{ version }}.tar.gz"
+  sha256: "cde644b272c2284f1212f0e0d25bdb9eaf7d58f1423ae82b712ba43db229d618"
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - dataclasses
+    - marshmallow >=3.0.0,<4.0
+    - marshmallow-enum
+    - pip
+    - python
+    - typeguard
+    - typing_inspect
+  run:
+    - dataclasses
+    - marshmallow >=3.0.0,<4.0
+    - marshmallow-enum
+    - python
+    - typeguard
+    - typing_inspect
+
+test:
+  imports:
+    - marshmallow_dataclass
+
+about:
+  home: "https://github.com/lovasoa/marshmallow_dataclass"
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: "Python library to convert dataclasses into marshmallow schemas."
+  doc_url: "https://lovasoa.github.io/marshmallow_dataclass/html/index.html"
+  dev_url: "https://github.com/lovasoa/marshmallow_dataclass"
+
+extra:
+  recipe-maintainers:
+    - lovasoa
```

Comment:
@lovasoa - are you willing to be a maintainer?

Reply:
Yes I do !

https://github.com/conda-forge/staged-recipes/pull/12730#issuecomment-698955408